index/ask/rpc/rpc.go

```diff
@@ -28,6 +28,7 @@ func (s *RPC) Get(ctx context.Context, req *GetRequest) (*GetResponse, error) {
 		storage[key] = &StorageAsk{
 			Price:        ask.Price,
```

Comment:
@asutula , I just did the usual RPC/CLI changes to be coherent.. but this will go away anyway in your changes.

Reply:
👍